views/projects.tpl

```diff
@@ -184,24 +184,28 @@ product_tags = {
     "technical": "Links to technical information"
 }
 
-categories = {
+unsorted_categories = {
     "Privacy": [0, "Privacy Protection & Cryptography"],
     "Blockchain": [1,"Blockchains & Smart Contracts"],
     "Verification": [2, "Software Verification"],
     "Security": [3, "Device and System Security"],
     "Learning": [4, "Machine Learning"],
-    "Other": [5, "Other"],
 }
 
-applications = {
+categories = dict(sorted(unsorted_categories.items(), key=lambda k: k[1][1]))
```

Comment:
do you know why the first field of the value (the "sort" number) is needed? can we get rid of it?

maybe @ineiti knows best why it was added

Reply:
The sort index is used down in the template to define the order of display for categories in the table